pkg/view/employee.go

```diff
@@ -27,6 +27,12 @@ type EmployeeData struct {
 	WorkingStatus model.WorkingStatus `json:"status"`
 	JoinedDate    *time.Time          `json:"joinedDate"`
 	LeftDate      *time.Time          `json:"leftDate"`
+
+	AccountStatus model.AccountStatus `json:"accountStatus"`
+	Seniority     string              `json:"seniority"`
+	Chapter       string              `json:"chapter"`
+	LineManager   string              `json:"lineManager"`
```

Comment:
This should be an object with ID Name and Avatar. and should be null if it's an empty object.